examples/fxa-geodb-example.js

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+var path = require('path');
+
+(function() {
+  'use strict';
```

Comment:
the example can be simplified:

1. remove `use strict`
2. remove function IIFE
3. the require statement should just point to https://github.com/TDA/fxa-geodb/blob/master/package.json#L6 without `path.join` using a relative path.